zigpy/zcl/clusters/general.py

```diff
@@ -915,31 +875,21 @@ class AttributeDefs(BaseAttributeDefs):
         reporting_status: Final = foundation.ZCL_REPORTING_STATUS_ATTR
 
     class ServerCommandDefs(BaseCommandDefs):
-        off: Final = ZCLCommandDef(
-            id=0x00, schema={}, direction=Direction.Client_to_Server
-        )
-        on: Final = ZCLCommandDef(
-            id=0x01, schema={}, direction=Direction.Client_to_Server
-        )
-        toggle: Final = ZCLCommandDef(
-            id=0x02, schema={}, direction=Direction.Client_to_Server
-        )
+        off: Final = ZCLCommandDef(id=0x00, schema={})
```

Comment:
As far as I understand, this should fix the issue I have seen in #1612: without detached mode, the change of the relay state wasn't reported correctly back to HASS?

Is my understanding correct?